<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>770</width>
        <height>640</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Compiler</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>728</width>
                <height>540</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="compiler_path_label">
                   <property name="text">
                    <string>Compiler path:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="compiler_path_input"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Compiler arguments:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="compiler_args_input"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>View</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>728</width>
                <height>540</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Editor font:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="editor_font_button">
                   <property name="text">
                    <string>Choose...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Shortcuts</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QScrollArea" name="scrollArea_4">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>728</width>
                <height>540</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QGroupBox" name="file_groupbox">
                 <property name="title">
                  <string>File</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Open</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QKeySequenceEdit" name="file_open_seq_edit"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Save</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QKeySequenceEdit" name="file_save_seq_edit"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="tab_groupbox">
                 <property name="title">
                  <string>Tab</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>New</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QKeySequenceEdit" name="tab_new_seq_edit"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Close</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QKeySequenceEdit" name="tab_close_seq_edit"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Next</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QKeySequenceEdit" name="tab_next_seq_edit"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Previous</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QKeySequenceEdit" name="tab_prev_seq_edit"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="folder_groupbox">
                 <property name="title">
                  <string>Folder</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Open</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QKeySequenceEdit" name="folder_open_seq_edit"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="run_groupbox">
                 <property name="title">
                  <string>Run</string>
                 </property>
                 <layout class="QFormLayout" name="formLayout_4">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Compile</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QKeySequenceEdit" name="run_compile_seq_edit"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Exec</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QKeySequenceEdit" name="run_exec_seq_edit"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>save_button</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
 </slots>
</ui>
